frontend/lib/src/components/core/LibContext.tsx

```diff
@@ -26,6 +26,7 @@ import { baseTheme, ThemeConfig } from "@streamlit/lib/src/theme"
  */
 export type LibConfig = {
   // TODO(lukasmasuch): Add lib config options in subsequent PRs.
+  userMapboxToken?: string
```

Comment:
This needs to be renamed since it is not necessarily the token from the user - but depending on the setup - the token configured by the platform: 

```suggestion
  mapboxToken?: string
```